pkg/services/ssosettings/ssosettingsimpl/service.go

```diff
@@ -130,6 +130,23 @@ func (s *SSOSettingsService) List(ctx context.Context) ([]*models.SSOSettings, e
 	return result, nil
 }
 
+func (s *SSOSettingsService) ListWithRedactedSecrets(ctx context.Context) ([]*models.SSOSettings, error) {
```

Comment:
Could we eventually re-use: https://github.com/grafana/grafana/blob/3d353a1b5f132f91db286dd7246133eb70a5ce26/pkg/setting/setting.go#L588

Reply:
I can update this PR to use `RedactedValue` , but we have a list of potential SSO settings secret names to add the list in `RedactedValue`? Like `client_secret`, etc. No problem if not.

Reply:
@gamab Thanks for pointing this out, we didn't know about this function. ❤️ 

@colin-stuart Yes, for now we only have `client_secret`, so please just add that! Thanks